recommender/static/html/recommender.html

```diff
@@ -129,109 +128,102 @@
                         <div class="recommender_clearFix"></div>
                         <div class='recommender_showProblematicReasons' style='display: none;' aria-hidden="true"></div>
                         <div class="recommender_clearFix"></div>
-                        <div class='recommender_descriptionText'> This is a list of resources your fellow students
-                            thought might be helpful. If you find another useful
-                            resource, either on edx.org or elsewhere, please add
-                            it. If you can improve the description or preview of a
-                            resource, please do so as well. If you find a resource
-                            helpful, upvote it. If it&rsquo;s not so helpful, downvote
-                            it. If it has issues (illegal material, incorrect, etc.),
-                            please flag it and let us know the reason. </div>
+                        <div class='recommender_descriptionText'>${_("This is a list of resources your fellow students thought might be helpful. If you find another useful resource, either on edx.org or elsewhere, please add it. If you can improve the description or preview of a resource, please do so as well. If you find a resource helpful, upvote it. If it&rsquo;s not so helpful, downvote it. If it has issues (illegal material, incorrect, etc.), please flag it and let us know the reason.")} </div>
                         <img class="recommender_previewingImg" style='display: none;' src="" aria-hidden="true"/>
                     </div>
                 </div>
             </div>
         </div>
         <div class='recommenderModify' style='display: none;' tabindex="0" role='region' aria-hidden="true">
             <div class='recommenderModifyTitleContainer'>
-                <div class='recommender_backToViewButton' role='button' tabindex="0" aria-label='cancel and go back to resource list'>&lt; Related resources</div>
+                <div class='recommender_backToViewButton' role='button' tabindex="0" aria-label='${_("cancel and go back to resource list")}'>&lt; ${_("Related resources")}</div>
```

Comment:
same as the comment on "Add new resource" about RTL languages. Should we have `&lt;` as part of the `gettext` string here?

Reply:
```suggestion
                <div class='recommender_backToViewButton' role='button' tabindex="0" aria-label='${_("cancel and go back to resource list")}'>${_("&lt; Related resources")}</div>
```